tests/test_admin.py

```diff
@@ -32,7 +32,10 @@ def test_create_repository(self):
             a.create(temp_path)
 
             # Do a read.
-            rc = svn.remote.RemoteClient('file://' + temp_path)
+            if temp_path[0] == '/':
```

Comment:
Can you place a comment indicating that this is the duck-typing approach to detecting Windows versus non-Windows?